src/chromium/interoperability-with-cpp/example-bindings.md

```diff
@@ -0,0 +1,37 @@
+# Example bindings
+
+cxx requires you to declare the whole C++/Rust boundary in one of your `.rs`
+files. For instance:
+
+```rust
+#[cxx::bridge]
+mod ffi {
+    extern "Rust" {
+        type MultiBuf;
+
+        fn next_chunk(buf: &mut MultiBuf) -> &[u8];
+    }
+
+    unsafe extern "C++" {
+        include!("cxx-demo/include/blobstore.h");
+
+        type BlobstoreClient;
+
+        fn new_blobstore_client() -> UniquePtr<BlobstoreClient>;
+        fn put(&self, parts: &mut MultiBuf) -> u64;
+    }
+}
+```
+
+<details>
+Point out:
+
+* Native support for C++'s `std::unique_ptr` in Rust
+* Native support for Rust slices in C++
```

Comment:
Question: Is it worth mentioning both `cxx`'s `rust::Slice<T>` and `//base/containers/span_rust.h`?